stubs/docopt/docopt.pyi

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Iterable, Optional
+
+__version__: str
+
+def docopt(
+    doc: str,
+    argv: Optional[Iterable[str]] = ...,
+    help: bool = ...,
+    version: Optional[Any] = ...,
```

Comment:
Maybe this should actually just be `Any`? Since `Any` matches against `None` as well?

Reply:
I'd say `Optional[Any]` is slightly better. It clearly says "this can be None".

Reply:
Sounds good, I lean that way slightly as well.